segmentation_models_pytorch/utils/functional.py

```diff
@@ -1,6 +1,24 @@
+import numpy as np
 import torch
 
 
+def revert_preprocessing_fn(x:np.ndarray, preprocessing_fn):
```

Comment:
What is the type of preprocessing function? Why it has a keyword attribute?

Reply:
This function is an aux function to view the images unnormelized by the model's preprocessing function (`smp.encoders.get_preprocessing_fn`). What do you mean by "keyword attribute"?